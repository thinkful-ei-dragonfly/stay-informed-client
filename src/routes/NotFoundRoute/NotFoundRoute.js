import React, { Component } from 'react'

class NotFoundRoute extends Component {
  render() {
    return (
      <section>
        <h2>404 - Page not found</h2>
        <p>Try going back to your previous page.</p>
      </section>
    );
  }
}

export default NotFoundRoute
