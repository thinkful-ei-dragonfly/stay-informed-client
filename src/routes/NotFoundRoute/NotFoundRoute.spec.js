import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NotFoundRoute from './NotFoundRoute'

describe(`NotFoundRoute component`, () => {
  it('renders a form.NotFoundRoute by default', () => {
    const wrapper = shallow(<NotFoundRoute />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
