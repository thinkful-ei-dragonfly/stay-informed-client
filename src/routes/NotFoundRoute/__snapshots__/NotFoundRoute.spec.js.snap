// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundRoute component renders a form.NotFoundRoute by default 1`] = `
<section>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
